Translation/Extractor/File/DefaultPhpFileExtractor.php

```diff
@@ -159,19 +159,36 @@ public function enterNode(Node $node)
         }
 
         $id = $node->args[0]->value->value;
-
         $index = $this->methodsToExtractFrom[strtolower($methodCallNodeName)];
-        if (isset($node->args[$index])) {
-            if ($node->args[$index]->value instanceof Node\Expr\ConstFetch && 'null' === (string) $node->args[$index]->value->name) {
+        $domainArg = null;
+
+        if (isset($node->args[$index]) && $node->args[$index] instanceof Node\Arg && null === $node->args[$index]->name ) {
+            $domainArg = $node->args[$index];
+        } else {
+            foreach ($node->args as $arg) {
+                if (!$arg instanceof Node\Arg) {
+                    continue;
+                }
+
+                if (null !== $arg->name && 'domain' === $arg->name->name) {
+                    $domainArg = $arg;
+
+                    break;
+                }
+            }
+        }
+
+        if (null !== $domainArg) {
+            if ($domainArg->value instanceof Node\Expr\ConstFetch && 'null' === (string) $domainArg->value->name) {
```

Comment:
`name` property for Argument was added in PHP-Parser 4.9.0.
Maybe it's judicious to bump the PHP-Parser version to `^4.9` in composer.json?